extras/03_nfs.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+
+set -eux
+
+figlet "Deploying nfs" | lolcat
+
+yum -y install nfs-utils
+mkdir /windows
+echo "/windows *(rw,no_root_squash)"  >>  /etc/exports
```

Comment:
We should avoid adding this entry twice:

```
grep -q '/windows' /etc/exports || \
    sudo echo "/windows *(rw,no_root_squash)"  >>  /etc/exports
```

Reply:
changed